setup/sql/dbupdate_custom.php

```diff
@@ -144,4 +144,16 @@
 <#13>
 <?php
 	$ilCtrlStructureReader->getStructure();
+?>
+
+<#14>
+<?php
+if($ilDB->tableColumnExists('usr_data', 'email')) {
+	$field = array(
+		"type" => "text",
+		"length" => 140,
+		"notnull" => false
```

Comment:
War es vorher möglich NULL einzutragen?

Reply:
Ich hab mir die Struktur in der Datenbank angesehen. Da steht unter "NULL" ein einfaches "Ja".

Reply:
OK!